specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-28-preview",
+    "title": "Customer Lockbox",
+    "description": "Azure Customer Lockbox API Reference"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/providers/Microsoft.CustomerLockbox/operations": {
+      "get": {
+        "tags": ["Operations"],
+        "description": "Lists all the available REST API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully retrieved operation list.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/OperationResult"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CustomerLockbox/requests/{requestId}": {
```

Comment:
I think in the past we talked about the creation flow for these requests. have you closed on those options? there might be scenarios where Lockbox request need to be auto created ( say Watson preapproval) and having a “CREATE” or PUT API might also help

Reply:
We are definitely thinking about these scenarios.